$NetBSD: patch-Config.mk.in,v 1.1 2019/09/16 11:42:50 nia Exp $

hunk 1:

Use BSD_INSTALL_*.

hunk 2:

Let pkgsrc set CFLAGS/LDFLAGS.

--- Config.mk.in.orig	2018-07-30 11:06:03.000000000 +0000
+++ Config.mk.in
@@ -9,11 +9,11 @@ VERSION		:= @PKG_VERSION@
 
 CC		:= @CC@
 MSGFMT		:= @MSGFMT@
-INSTALL		:= @INSTALL@
+INSTALL		:= ${BSD_INSTALL}
 
-INSTALLEXE	:= ${INSTALL} -D -p -m 755 -s
-INSTALLSCR	:= ${INSTALL} -D -p -m 755
-INSTALLDATA	:= ${INSTALL} -D -p -m 644
+INSTALLEXE	:= ${BSD_INSTALL_PROGRAM}
+INSTALLSCR	:= ${BSD_INSTALL_SCRIPT}
+INSTALLDATA	:= ${BSD_INSTALL_DATA}
 
 ################ Destination #########################################
 
@@ -27,13 +27,12 @@ O		:= .o/
 
 ################ Compiler options ####################################
 
-CFLAGS		:= -Wall -Wextra -Wredundant-decls -Wshadow \
+CFLAGS		+= -Wall -Wextra -Wredundant-decls -Wshadow \
 		   -std=c11 -I/usr/include/libxml2
-LDFLAGS		:= -liconv -lintl -lxml2 -lncurses -lz
+LDFLAGS		+= -liconv -lintl -lxml2 -lncurses -lz
 ifdef DEBUG
     CFLAGS	+= -O0 -ggdb3
     LDFLAGS	+= -g -rdynamic
 else
-    CFLAGS	+= -Os -g0 -DNDEBUG=1
-    LDFLAGS	+= -s
+    CFLAGS	+= -DNDEBUG=1
 endif
